jobs/build/gen-assembly/Jenkinsfile

```diff
@@ -39,7 +39,7 @@ node {
                         ),
                         string(
                             name: "NIGHTLIES",
-                            description: "(Optional) List of nightlies to match with <code>doozer get-nightlies</code> (if empty, find latest)",
+                            description: "(Optional for public nightlies) List of nightlies to match with <code>doozer get-nightlies</code> (if empty, find latest). If preparing from private nightlies, provide the amd64 nightly as parameter.",
```

Comment:
why only amd64? I think it should be "provide all nightlies for desired arches"

Reply:
the code will auto fetch the other arches.

Reply:
ah. Then I'd suggest "provide the amd64 nightly to match"

Reply:
updated